src/praisonai/tests/unit/test_openai_refactor.py

```
@@ -0,0 +1,219 @@
+
+def test_openai_client():
+    """Test the OpenAI client module"""
+    print("Testing OpenAI Client Module...")
+    
+    try:
+        # Test 1: Get global client
+        client = get_openai_client()
+        print("✅ Global OpenAI client created successfully")
+        
+        # Test 2: Create new client instance
+        new_client = OpenAIClient()
```

Ruff (0.11.9)

Local variable `new_client` is assigned to but never used

Remove assignment to unused variable `new_client`

(F841)